src/store/migrations/tests/password.test.js

```diff
@@ -0,0 +1,101 @@
+// @flow strict
+
+import assert from 'assert'
+
+import { WALLETS } from './data'
+
+import {
+  migratePassword,
+  checkPasswordMigrationV1Needed,
+} from '../password'
+
+const RESULT = {
+  version: 1,
+  internalKey: WALLETS.v0.internalKey,
+  salt: WALLETS.v0.passwordOptions.salt,
+  hint: WALLETS.v0.passwordOptions.passwordHint,
+}
+
+const RESULT_EMPTY = {
+  salt: '',
+  hint: '',
+  version: 1,
+  internalKey: null,
+}
+
+describe('store/migrations/password', () => {
+  test('should check if migration to v1 is needed', async () => {
```

Comment:
Oh, we should do a meetup on testing styles (behaviour / assert / ...)! And also decide on testing style for Wallet – I did it in another one, so I will need to refactor those a little to be consistent with yours.